src/multirust-cli/rustup_mode.rs

```diff
@@ -374,3 +383,9 @@ fn self_uninstall(m: &ArgMatches) -> Result<()> {
 
     self_update::uninstall(no_prompt)
 }
+
+fn telemetry(cfg: &Cfg, m: &ArgMatches) -> Result<()> {
+    let telemetry_string = m.value_of("enabled").unwrap();
+    
+    cfg.set_telemetry(telemetry_string)
```

Comment:
Makes sense - there's less chance of writing "potato" to a telemetry file this way.